CHANGELOG.md

```diff
@@ -19,12 +26,13 @@
 #### Test improvements
 
 - A new script to run test on AWS makes it easier to test under several distributions: CentOS, Debian, Fedora, Ubuntu ([#1925](https://github.com/coreos/rkt/pull/1925)).
-- The functional tests now skip user namespace tests when user namespace does not work([#1947](https://github.com/coreos/rkt/pull/1947)).
+- The functional tests now skip user namespace tests when user namespaces do not work ([#1947](https://github.com/coreos/rkt/pull/1947)).
+- Check we're not building rkt with go 1.5.{0,1,2} to make sure we're not vulnerable to CVE-2015-8618 ([#2006](https://github.com/coreos/rkt/pull/2006)).
```

Comment:
Avoid "we":

Check rkt is not build with...

...to make sure rkt is not vulnerable...